content/blog/configuring-sso-for-hpe-greenlake-central-private-cloud-enterprise-and-hpe-greenlake-glcp-using-okta.md

```diff
@@ -0,0 +1,162 @@
+---
+title: How to implement a single sign-on solution to authenticate users onto the
+  HPE GreenLake edge-to-cloud platform
+date: 2023-11-29T11:34:12.167Z
+featuredBlog: false
+externalLink: ""
+author: Prabhu Murthy - Cloud Operations
+authorimage: /img/Avatar1.svg
+thumbnailimage: /img/hpe-greenlake-sso-page.jpg
+disable: false
+tags:
+  - HPE GreenLake Private Cloud Enterprise application
+  - HPE GreenLake edge-to-cloud platform
+  - sso
+  - okta
+  - Active Directory
+  - hpe-greenlake
+---
+Enterprises looking to use HPE GreenLake for Private Cloud Enterprise can benefit from the use of SSO, as it has been integrated onto the HPE GreenLake edge-to-cloud platform, which supports single sign-on
+
+In this blog post, I will walk you through the process of configuring Okta Active Directory (AD) to authenticate users into the HPE GreenLake for Private Cloud Enterprise application on the HPE GreenLake platform using SAML Identity Provider (IdP) for single sign-on.
+
+### Before starting
+
+Please review the [HPE GreenLake](https://support.hpe.com/hpesc/public/docDisplay?docId=a00120892en_us&page=GUID-D7192971-EF71-4304-B51E-548E7954E644.html) User Guide to understand how the SAML framework works in the context of Common Cloud Services for the HPE GreenLake edge-to-cloud platform for the HPE GreenLake Private Cloud Enterprise application .
```

Comment:
You may want to add a note about what is Common Cloud Services to set the context. Readers will not know about CCS.

Also, throughout the blog post, you should use "HPE GreenLake for Private Cloud Enterprise".  (notice the "for" after HPE GreenLake and before Private Cloud Enterprise).